### PR TITLE
Add Joke Component in Header

### PR DESCRIPTION
**Pull request by knkp**
Created a new component `<JokeQuote/>` which grabs a random text response from jokeapi.dev and set's it to be random quote of a random smart person.

i'M wOrKiNg

Favorite one during testing (paraphrased):

```
" You have reached the rate limit for the public api "
- Mark Zuckerberg
```

### CODE REVIEW

File: src/Home/JokeQuote.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "Tim Berners-Lee",
+    "Hedy Lamarr",
+    "Bill Gates"
+];
+
+function JokeQuote()
+{
+	const [gotJoke, setGotJoke] = useState(false);
+	const [joke, setJoke] = useState("");
+	const [peep, setPeep] = useState("");
```
Comment (ted-marozzi) [inline src/Home/JokeQuote.tsx:22]: Your naming conventions is horrible lol "peep", "smarty"

File: src/Home/JokeQuote.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+			</div>
+			<div>
+				- {peep}
+			</div>
+		</>
+	}		
+	</>);
+}
+
+export default JokeQuote;
```
Comment (ted-marozzi) [inline src/Home/JokeQuote.tsx:71]:
```suggestion
export default JokeQuote;

```
New line at end of file

**Review by ted-marozzi — APPROVED** [review]
Fix the naming convention if you feel like, but great stuff, your contribution is highly valued

File: src/Home/JokeQuote.tsx (hunk 1 above)
Comment (knkp) [reply src/Home/JokeQuote.tsx:22]: My bad, probably spent too much time in the facebook group before building this lol, I'll issue another pull request sometime this week with a better naming convention.

**Pull request merged**